typing/tast_iterator.ml

```diff
@@ -548,6 +548,7 @@ let typ sub {ctyp_loc; ctyp_desc; ctyp_env; ctyp_attributes; _} =
   | Ttyp_variant (list, _, _) -> List.iter (sub.row_field sub) list
   | Ttyp_poly (_, ct) -> sub.typ sub ct
   | Ttyp_package pack -> sub.package_type sub pack
+  | Ttyp_open (_, _, t) -> sub.typ sub t
```

Comment:
We probably want some `iter_loc` call in this case as well.

Reply:
[393c0e7](https://github.com/ocaml/ocaml/pull/12044/commits/393c0e7171baf3045f3725c3fd230e4276012bf3)